### DISCUSSION TITLE
Invalidate cache from specific folder in node_modules

### DISCUSSION THREAD

**Discussion by Alpacora** [vercel/next.js#54801] [Help]
### Summary

Hi guys, thanks for joining this trend. I will try soon.

I'm looking for ways to invalidate the cache of a specific folder inside node_modules so that every hot reload the files in that folder are transpiled again.

Let me put you in context, I have a component library in a github repository, I install it through SSH without doing any builds, a third-party library like the tailwind css calls its documentation.

But I would like that when I change the .tsx files manually inside the node_modules in my library folder the cache is reset and the changes are applied.

I managed to make this work in a CRA project using a library react app rewire but I couldn't do it in nextjs, does anyone have any suggestions?

### Example

This is my current next.config.js file 

<img width="415" alt="image" src="https://github.com/vercel/next.js/assets/38174807/510cc8d2-cff6-468a-822a-4699efd3e4a7">

My tailwind.config.ts file 

<img width="610" alt="image" src="https://github.com/vercel/next.js/assets/38174807/6b2c97ad-dba6-481f-8780-f8bbc1a724d4">

I keep thinking that `transpilePackages: ["@epione/epione-components"]` should have solved the problem I mentioned but ended up solving the error of running on the client side.
_No response_

**Comment by jvitoralmeida01** [comment]
I'm facing the exact same problem rn... Would be great to get some insight on this 🙏
